Helper/Account.php

```diff
@@ -107,7 +104,7 @@ public function __construct(
         $this->logger = $context->getLogger();
         $this->nostoHelperScope = $nostoHelperScope;
         $this->nostoHelperUrl = $nostoHelperUrl;
-        $this->urlBuilder = $urlBuilder;
+        $this->urlBuilder = $this->_urlBuilder;
```

Comment:
Why not remove this and use the parent `_urlBuilder`? Doesn't look like you modify anything in the parent var anyways.